src/shogun/evaluation/ClusteringAccuracy.cpp

```diff
@@ -23,6 +23,9 @@ float64_t CClusteringAccuracy::evaluate(CLabels* predicted, CLabels* ground_trut
 	int32_t correct=0;
 	for (int32_t i=0; i < predicted_ilabels.vlen; ++i)
 	{
+		/* print progress */
+		SG_PROGRESS(i, 0, predicted_ilabels.vlen, 1, "EVALUATING ACCURACY: ");
```

Comment:
By _vectorized_ you mean "vectorized by" the compiler (using something like `-ftree-vectorize`), right? I understand your point. Maybe the progress bar can be more useful on longer evaluation methods (I suppose the only candidate here is cross-validation).